vdb-bench-assembly/dev-server/json/vdbs.json

```diff
@@ -1,15 +1,15 @@
 {
     "vdbs": [
         {
-            "id": "portfolio",
+            "id": "Portfolio",
```

Comment:
I think a "description" would be nice here also. I kind of remember you having it before.